neo/Wallets/Wallet.py

```diff
@@ -1054,6 +1054,14 @@ def MakeTransaction(self,
 
         self._vin_exclude = None
 
+        synced_state = self.GetSyncedState()
+
+        if not synced_state:
+            logger.warn("Wait for your wallet to be synced before doing \
```

Comment:
Don't add line breaks, please use this syntax for long strings:
 
    s = ("this is a very"
         "long string too"
         "for sure ...")